nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -112,6 +112,19 @@
       </link>
     </para>
    </listitem>
+   <listitem>
+     <para>
+     IPv6 Privacy Extensions are now enabled by default for undeclared
+     interfaces. The previous behaviour was quite misleading — even though
+     the default value for
+     <option>networking.interfaces.*.preferTempAddress</option> was
```

Comment:
```suggestion
     <option>networking.interfaces.<name?>.preferTempAddress</option> was
```

Reply:
That's not valid XML AFAIK. I'd prefer leaving it as * to improve readability of the source.

Reply:
I was trying to make it appear the same as in the option search https://nixos.org/nixos/options.html#networking.interfaces.+prefertempaddress